test/Libraries/RevitNodesTests/Elements/TagTests.cs

```diff
@@ -30,9 +30,50 @@ public void ByElement_ValidArgs()
 
             var tag = Tag.ByElement(Revit.Application.Document.Current.ActiveView, wall, true, false, "Center", "Middle", Vector.ByCoordinates(0, 0, 0));
             Assert.NotNull(tag);
+
+            tag = Tag.ByElementAndLocation(Revit.Application.Document.Current.ActiveView, wall, Point.ByCoordinates(0,0,0));
```

Comment:
in these tests can you also check the resulting position of the tag is correct and that the internal element is actually a non null tag? please do this for all tests. Just asserting the wrapper is not null is not sufficient.